x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rules_list.tsx

```diff
@@ -286,6 +329,30 @@ export const RulesList: React.FunctionComponent = () => {
     })();
   }, []);
 
+  useEffect(() => {
+    if (setStatusFilter) {
```

Comment:
I think we should rename these to `onStatusFilterChange`, to differentiate it from local setStates. When I first read this I was confused at why we're checking if setState exists haha